File: specification/specs/x_pack/machine_learning/stop_datafeed/StopDatafeedRequest.ts
@rest_spec_name('ml.stop_datafeed')
class StopDatafeedRequest extends RequestBase {
  path_parts?: {
    datafeed_id: Id
  }
  query_parameters?: {
    allow_no_datafeeds?: boolean
  }
  body?: {
    force?: boolean
    timeout?: Time
  }
}

File: output/csharp/XPack/MachineLearning/StopDatafeed/StopDatafeedRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class StopDatafeedRequest : RequestBase {
		
		[DataMember(Name="allow_no_datafeeds")]
		public bool AllowNoDatafeeds { get; set; }


		[DataMember(Name="force")]
		public bool Force { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }

	}
}
